tests/test_superruns.py

```diff
@@ -0,0 +1,247 @@
+import os
+import unittest
+import shutil
+import strax
+import numpy as np
+import tempfile
+from strax.testutils import Records, PeaksWoByRunDefault
+import datetime
+import pytz
+import json
+from bson import json_util
+import re
+
+
+class TestSuperRuns(unittest.TestCase):
+
+    def setUp(self, superrun_name='_superrun_test'):
+        self.offset_between_subruns = 10
+        self.superrun_name = superrun_name
+        self.tempdir = tempfile.mkdtemp()
```

Comment:
I assume you use this tempfile since you want to re-use the results and not start making records from scratch. Perhaps this is worth commenting on